media-libs/opencolorio/patches/opencolorio-2.3.2.patchset

```diff
@@ -0,0 +1,31 @@
+From 2521e7480806b0b374318ef11cfff2ea3772236e Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Tue, 18 Jun 2024 19:06:09 +0200
+Subject: Fix build (hackish)
+
+
+diff --git a/src/utils/NumberUtils.h b/src/utils/NumberUtils.h
+index 08a9ba0..8b0ec8d 100644
+--- a/src/utils/NumberUtils.h
++++ b/src/utils/NumberUtils.h
+@@ -65,7 +65,7 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
+ #ifdef _WIN32
+     tempval = _strtod_l(first, &endptr, loc.local);
+ #else
+-    tempval = ::strtod_l(first, &endptr, loc.local);
++    tempval = ::strtod(first, &endptr);
```

Comment:
Not sure it's correct, been strugling with it and couldn't find a descent sollution so far. Help wanted. :)

Reply:
strtod_l is available under `__USE_GNU`, it seems: https://xref.landonf.org/source/xref/haiku/src/system/libroot/posix/glibc/stdlib/stdlib.h#249

I don't remember exactly how to get that though. Shouldn't that be the default? Does the CMake file set the C++ standard explicitly to override that maybe?

Reply:
My bad, this is an internal header in libroot, so it probably isn't available.

Reply:
Have been searching the function in the headers ... couldn't find it (not even in the source for Haiku).

Reply:
Updated patchset to be a bit more inline with OS checks.

Reply:
Updated here to be inline with the other one.